IndomitableIbises/Bangazon-Sprint-3
# Nav Updated/Payment Button

## Link to Ticket
Closes # [Link]()

## Description of Proposed Changes

- Linked Profile, Order History, and Shopping Cart
- User can now view their payments and add a payment option
- Extended index to all templates

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout RC-NavUpdate
```
1. Close Server and DB Browser
1. `./django_data.sh website faker_factory` or `django_data.sh website faker_factory`
1. `python manage.py runserver`
1. Register a new user
1. Open DB Browser and manually add orders and assign the new user ID where appropriate
1. Navigate to site and check Profile, Order History, Shopping Cart, Payments
1. Click **Add Payment** button and then view it in the payment list
1. Nav Bar should be showing up in all logged in views
1. Raise the roof




## Impacted Areas in Application

List general components of the application that this PR will affect:

* all templates.html

## Mentions @username

Tag users that need to review this code
@sirwin4 @paulzimmclay 
## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
